raiden/ui/cli.py

```diff
@@ -204,23 +205,35 @@ def app(address,
 
         address = addresses[idx]
 
-    unlock_tries = 3
-    while True:
+    password = os.environ.get('ETH_PASSWORD')
```

Comment:
Yeah having the env var option might be a bit too much of a foot-gun.
A possible compromise which keeps passwords out of the shell history but still allows scriptability:
Add a `--password-file` option which would read the password from the file provided.